devenv.nix

```diff
@@ -286,4 +287,55 @@ in {
   scripts.debug.exec = ''
     XDEBUG_SESSION=1 ${phpXdebug}/bin/php "$@"
   '';
+
+  scripts.importdb.exec = ''
+    if [[ "${config.env.DATABASE_DUMP_URL}" == "" ]]; then
+      echo "Please setup ENV DATABASE_DUMP_URL"
+      exit
+    fi
+
+    echo "Are you sure you want to download the file and overwritting database shopware with its data (y/n)?"
+    read answer
+
+    if [[ "$answer" != "y" ]]; then
+        echo "Allright, we will stop here."
+        exit
+    fi
+
+    BASENAME=$(basename -- ${config.env.DATABASE_DUMP_URL})
```

Comment:
do you need a basename when analyzing the used extension? if not this can just be removed :)

Reply:
Oh yeah, we can save this variable. Will be removed